lib/_layouts/_base.njk

```diff
@@ -27,6 +28,18 @@
   </title>
   {% endblock %}
 
+  {% block googleAnalytics %}
+    <!-- Google tag (gtag.js) -->
+    <script async="async" src="https://www.googletagmanager.com/gtag/js?id=G-NQ562NM01Z" type="text/plain" data-cookieconsent="statistics"></script>
```

Comment:
I know that access to GA may be restricted but it could be an option to move out `gtag` so that its an environment variable in Github? That said, I don't know how often if at all it would ever need to be updated.

Reply:
Its unlikely to change. I think there's an argument to choose different values per env, but we only have the one. So for simplicity, can we keep as is?